workflows/models.py

```diff
@@ -513,7 +513,7 @@ def _create_for_analyses(
         """
         logger.info(f"Creating batch for {len(analysis_ids)} analyses")
 
-        batch, _ = AssemblyAnalysisBatch.objects.get_or_create(
+        batch = AssemblyAnalysisBatch(
             study=study,
             batch_type=StudyAnalysisBatch.BatchType.ASSEMBLY_ANALYSIS,
```

Comment:
random side point, but this type should probably be defined as a class var like `AssemblyAnalysisBatch.BATCH_TYPE` or something. I feel like it belong to the specific batch type class not the generic.. maybe.